demo/index.html

```diff
@@ -7,18 +7,21 @@
     <link
       rel="stylesheet"
       type="text/css"
-      href="../node_modules/prismjs/themes/prism.css"
+      href="https://cdn.jsdelivr.net/npm/prismjs@1.24.1/themes/prism.css"
```

Comment:
What was the reason for locking this to this version rather than using latest like auro-input and other components?
```suggestion
      href="https://cdn.jsdelivr.net/npm/prismjs@latest/themes/prism.css"
```

Reply:
@blackfalcon , could you please explain as this code was changed by you?